src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,363 @@
+        if (split == null) {
+            customPrint("I could not find '/by' in your command!");
+        } else if (split[0].equals("")) {
+            customPrint("I could not find the title of your deadline!");
+        } else {
+            String taskDescription = split[0];
+            String date = split[1];
+            tasks.add(new Deadline(taskDescription, date));
+            acknowledgeCommand();
+        }
+    }
+
+    /**
+     * Adds an Event to the Task list
+     *
+     * @param description Task description entered by user.
+     */
+    private static void addEvent(String description) {
+        String[] split = spiltString("/at", description);
+        // Handle case it user did not provide date
+        if (split == null) {
+            customPrint("I could not find '/at' in your command!");
+        } else if (split[0].equals("")) {
+            customPrint("I could not find the title of your event!");
+        } else {
+            String taskDescription = split[0];
+            String at = split[1];
+            tasks.add(new Event(taskDescription, at));
+            acknowledgeCommand();
+        }
+    }
+
+    /**
+     * Removes a Task from the Task list
+     *
+     * @param id Task id entered by user.
+     */
+    private static void deleteTask(String id) {
+        if (!checkValidTaskId(id)) { // Invalid task id
+            return;
+        }
+
+        int taskId = Integer.parseInt(id) - 1;
+        Task removedTask = tasks.get(taskId);
+        tasks.remove(taskId);
+        customPrint("Noted. I've remove this task:\n" + removedTask.toString() + "\nNow you have "
+                + tasks.size() + " tasks in the list.");
+    }
+
+    /**
+     * Prints the exit message and ends the program.
+     */
+    private static void exit() {
+        customPrint("Bye. Hope to see you again soon!");
+        // Ends the program
+        System.exit(0);
+    }
+
+    /**
+     * Prints the acknowledgement that the command is processed.
+     */
+    private static void acknowledgeCommand() {
+        Task addedTask = tasks.get(tasks.size() - 1);
+        customPrint("Got it. I've added this task:\n" + addedTask.toString() + "\nNow you have "
+                + tasks.size() + " tasks in the list.");
+    }
+
+    /**
+     * Prints the welcome message.
+     */
+    private static void printWelcomeMessage() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        // Prints welcome message
+        String welcomeMessage = logo;
+        welcomeMessage += "Hello! I'm Duke\n";
+        welcomeMessage += "What can I do for you?";
+        customPrint(welcomeMessage);
+    }
+
+    /**
+     * Prints the invalid command message.
+     */
+    private static void printInvalidCommand() {
```

Comment:
May want to rename this method, since it is not printing the invalid command, but the invalid command message.